### PR TITLE
Create experiment endpoints tests

### PR DESCRIPTION
**Pull request by erichanslee**
Tests pulled from `test_driver.py`, `test_model.py`, and then I added a few more

### CODE REVIEW

**Review by harvey-intel — APPROVED** [review]
Mostly LGTM, a few more nits.

File: sigoptlite/sigoptlitetest/endpoints/test_suggestion.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    s = self.conn.experiments(e.id).suggestions().create()
+    with pytest.raises(ValueError) as exception_info:
+      self.conn.experiments(e.id).observations().create(
+        suggestion=s.id + "1",
+        values=[{"name": "y1", "value": 1}],
+      )
+    msg = f"The suggestion you provided: {s.id + '1'} does not match the suggestion stored: {s.id}"
+    assert exception_info.value.args[0] == msg
+
+  def test_no_suggestion_id(self, experiment_meta):
```
[the remark this answers is not in the document]
Comment (erichanslee) [reply sigoptlite/sigoptlitetest/endpoints/test_suggestion.py:33]: deleted

File: sigoptlite/sigoptlitetest/endpoints/test_experiment.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+
+class TestExperiment(UnitTestsBase):
+  conn = Connection(driver=LocalDriver)
+
+  @pytest.fixture
+  def experiment_meta(self):
+    return self.get_experiment_feature("default")
+
+  def test_driver_basic(self, experiment_meta):
```
Comment (malkomes) [inline sigoptlite/sigoptlitetest/endpoints/test_experiment.py:20]: experiment?

Comment (erichanslee) [reply sigoptlite/sigoptlitetest/endpoints/test_experiment.py:20]: ah whoops... thanks!

**Pull request merged**